secedgar/filings/combo.py

```diff
@@ -0,0 +1,106 @@
+from datetime import date, timedelta
+
+from secedgar.filings.daily import DailyFilings
+from secedgar.filings.quarterly import QuarterlyFilings
+from secedgar.utils import get_month, get_quarter, add_quarter
+
+
+class ComboFilings:
+    """Class for retrieving all filings between specified dates.
+
+    .. versionadded:: 0.4.0
+    """
+    def __init__(self, start_date: date, end_date: date, client=None,
+                 entry_filter=lambda _: True, balancing_point=30):
+        self.entry_filter = entry_filter
+        self.start_date = start_date
+        self.end_date = end_date
+        self.master = QuarterlyFilings(year=self.start_date.year, quarter=get_quarter(
```

Comment:
useful if we make other methods later

Reply:
Like what methods?

Reply:
Hey @reteps how would this be useful? I am currently of the opinion that we should just create `QuarterlyFilings` and `DailyFilings` only when `save` is called. Thoughts?

Reply:
Cons of doing it this way:

+ Computer has to store reference to 2 objects in memory
+ Possibly a little more complex to understand

Pros of doing it this way:

+ Future expandability of this class
+ You mentioned `which methods`, maybe a function that just spits out the URLs, or a function that downloads the index files instead of the `.txt` files, etc.

Let me know your thoughts, but I feel pretty strongly that putting these inside the class is a good move (though I have limited software development experience from my time in high school lol).